app/views/welcome/index.html.erb

```diff
@@ -21,4 +30,33 @@
   </div>
 </div>
 
+<script type="text/javascript">
+  $(document).ready(function(){
+    $(".overlay-arrow").click(function(){
+      $("#overlay").css({
+        "margin-top" : "-100vh"
+      });
+    });
+    $(window).on('scroll', function() {
+      var scroll = $(this).scrollTop();
+      console.log(scroll);
+      var height = window.innerHeight;             
+       if(scroll > 100) {
+  
+        $("#overlay").css({
+          "margin-top" : "-100vh"
+        })
+      }else if(scroll <= 1){
+        $("#overlay").css({
+          "margin-top" : "0vh"
+        })
+      }
+     });
+
```

Comment:
This is a CSS animation. We are looking for a wipe. Here's an example of what section wipes look like: http://janpaepke.github.io/ScrollMagic/examples/basic/section_wipes_natural.html